app/app.js

```diff
@@ -0,0 +1,31 @@
+const express = require('express');
+const path = require('path');
+const app = express();
+
+app.use(express.static(path.join(__dirname, '..', 'dist')));
+
+app.get('/preferences.js', function (req, res) {
+  api_url = process.env.REACT_APP_API_URL
+  login_page = process.env.REACT_APP_LOGIN_PAGE_URL
```

Comment:
We definitely want to check that the variable exists before setting it. Otherwise this code could set some config vars to "undefined".